### ISSUE TITLE
Drag & Drop crashes application

### ISSUE THREAD

**Issue by gingters** [EDCD/coriolis#98]
Whenever I want to drag & drop a component to another slot, the app crashes.
Happens on Edge on Win10 creators update.

Details:

Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36 Edge/15.15063
Path: /outfit/python
Error: Unknown
Details:
scriptUrl: https://coriolis.edcd.io/app.5ce9a6.js
line: 33
col: 20909
error: {}

**Comment by cmdrmcdonald** [comment]
Thank you for reporting this.  Does this happen with other browsers as well?  What exactly did you drag to cause the crash?  If you could provide the URL prior to the crash and details of exactly which module you dragged where we can try to track this down.

**Comment by gingters** [comment]
Hiho. I just re-tested.

The problem does not occur in Chrome of Firefox. I could reproduce it with the latest 2.3.3 - 2017-04-15 version of EDCD Coriolis in Microsoft Edge (40.15063.0.0) and in Microsoft Internet Explorer 11 (11.0.15063.0), Edge being my main browser.

Steps to reproduce:
Just open the app in IE or Edge, select any ship from the list, and start dragging any of the initial/default Optional internal or Hardpoints boxes. As soon as you move the box, the crash occurs.

I have a little bit of additional info as IE 11 shows the stack trace (while Edge sadly doesn't):

```
scriptUrl: https://coriolis.edcd.io/app.dd0d2b.js
line: 33
col: 20991
error: {"description":"Invalid argument.","number":-2147024809,"stack":"Error: Invalid argument.
   at value (https://coriolis.edcd.io/app.dd0d2b.js:49:18432)
   at r (https://coriolis.edcd.io/app.dd0d2b.js:33:20845)
   at o (https://coriolis.edcd.io/app.dd0d2b.js:33:16924)
   at s (https://coriolis.edcd.io/app.dd0d2b.js:33:17141)
   at f (https://coriolis.edcd.io/app.dd0d2b.js:24:11787)
   at m (https://coriolis.edcd.io/app.dd0d2b.js:24:11908)
   at r (https://coriolis.edcd.io/app.dd0d2b.js:49:1983)
   at y.processEventQueue (https://coriolis.edcd.io/app.dd0d2b.js:24:12993)
   at r (https://coriolis.edcd.io/app.dd0d2b.js:54:12378)
   at a.handleTopLevel (https://coriolis.edcd.io/app.dd0d2b.js:54:12481)"}
```

**Comment by cmdrmcdonald** [comment]
This is now fixed with the release of Coriolis 2.3.4.
